client/src/templates/BaseTemplate/style.scss

```diff
@@ -0,0 +1,12 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 8px;
+
+  .logo, .github-logo {
+    cursor: pointer;
+  }
+  img {
+    height: 25px;
```

Comment:
`.header img` is overriding `.logo-img`, not sure if this was your intention, if it was then you could probably remove `.logo-img`

![image](https://user-images.githubusercontent.com/9146455/197886580-7d38fc44-39ba-45d1-afb0-c10963909932.png)

Reply:
Good catch

Reply:
Nvm it's still being used in the room template.